css/src/components/form/textarea.scss

```diff
@@ -145,4 +148,15 @@ $textarea-divider-radius: 0.5rem !default;
 		gap: $textarea-form-gap;
 		height: 100%;
 	}
+
+	.textarea-form-help {
```

Comment:
Should the inline-start padding be on this element to ensure we don't need to add a margin atomic on the inner item each time?

<img width="284" alt="image" src="https://github.com/user-attachments/assets/b5b198a2-b69a-4cf1-8320-98474e83a74f" />

Reply:
changed